src/api/controllers/pets.controller.ts

```diff
@@ -1,10 +1,12 @@
 import express from "express";
-import { getDatabase } from "../utils/database";
-import { authToken } from "../utils/config";
+import { Request, Response } from "express";
+import { authToken } from "utils/config";
```

Comment:
Can make the import paths relative? `../../utils/config` instead of `utils/config`

Reply:
ok